addons/mail/static/src/discuss/core/common/channel_invitation.js

```diff
@@ -53,7 +53,7 @@ export class ChannelInvitation extends Component {
         });
         useEffect(
             () => {
-                if (this.props.autofocus) {
+                if (this.store.self.type === "partner" && this.props.autofocus) {
```

Comment:
It would be best to simply guard `this.inputRef.el` instead, the condition on partner comes out of nowhere.